Resources/doc/reference/installation.rst

```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle
+    $ composer require sonata-project/media-bundle
```

Comment:
> It would be great, but how?

Maybe something like adding search and replace on specific files (*.rst), which result in a PR on this repo IF there is a match?

In this case we could search for: `$ composer.phar` and replace it with `composer` in `*.rst` files?

> It is not a big deal indeed, but the $ means a prompt. A prompt is not always a $.

True, but this should be executed in userland and not in `#`

> For easy copy/paste, only the command should be here IMHO.

I agree with you, but the work is not worth to do it